src/store/store.ts

```diff
@@ -25,11 +22,6 @@ export const store = configureStore({
     getDefaultMiddleware().concat([aws.middleware, apes.middleware]),
 });
 
-config.oauth.redirectSignIn =
-  window.location.origin + `${appRoutes.auth_redirector}/`;
-config.oauth.redirectSignOut = window.location.origin + "/";
-Amplify.configure(config);
```

Comment:
@SovereignAndrey, `Amplify` need to be configured first before being used (required by `<Authenticator/>` component )